playbooks/install_os.yml

```diff
@@ -17,10 +28,25 @@
         tasks_from: install.yml
       tags:
         - edpm_sshd
-    - name: Install edpm_chrony
-      ansible.builtin.import_role:
-        name: osp.edpm.edpm_chrony
-        tasks_from: install.yml
+    - name: Set FQCN for the timesync role
+      ansible.builtin.set_fact:
+        timesyncfqn:
+          "{%- if ansible_facts['distribution'] == 'RedHat' -%}
+            redhat.rhel{%- else -%}
+            fedora.linux{%- endif -%}
+          _system_roles.timesync"
+      delegate_to: localhost
+    - name: Compile list of ntp servers with their attributes
+      ansible.builtin.set_fact:
+        timesync_ntp_servers: "{{ timesync_ntp_servers | default([]) + [{ 'hostname' : item }]}}"
+      loop: "{{ timesync_ntp_servers }}"
+      when: item is string
+      failed_when: >
+        ( item is hash ) and
+        ( 'hostname' not in item )
+    - name: Install and configure edpm_chrony
```

Comment:
Does edpm_chrony need to be changed here?

Reply:
Eh. I'm not really sure what should we call that step. Yes, it isn't the `edpm_chrony` role anymore. OTOH, the service is still for edpm and it's still chrony being deployed ... How about "Install and configure chrony"?

Reply:
What about "Install and configure chrony via linux-system-roles" ?

It could be helpful to declare that we are going to use a role from another collection. WDYT?

Reply:
I don't know about that, after all if they read the source they can see what we are using, and if something goes wrong they will see the role name in logs.

Reply:
I've changed the name.